02 Toolbar/src/editor.tsx

```diff
@@ -1,104 +1,80 @@
 import * as React from 'react';
 declare function require(name:string);
 const MTRC = require('markdown-to-react-components');
+import {textAreaManager} from './common/managers/textAreaManager';
 
 interface Props {
 }
 
 interface State {
-  content : string;
+  editorContent: string;
+  viewerContent: string;
+  shouldUpdateCursor: boolean;
 }
 
 export class EditorComponent extends React.Component<Props, State> {
-
-  textarea : any;
+  textArea : HTMLTextAreaElement;
+  offset: number;
+  cursorPosition: number;
 
   constructor(props: Props) {
     super(props);
 
-    this.state = {content: 'test'};
+    this.state = {
+      editorContent: 'test',
+      viewerContent: 'test',
```

Comment:
Add some simple markdown content for the sample (just to get something already showing)